shm/cargo.cpp

```diff
@@ -0,0 +1,15 @@
+#include "cargo.hpp"
+
+Cargo();
```

Comment:
I think it will not complie, there should be scope, like this:
Cargo::Cargo();

Reply:
But I think we use Cargo::Cargo(); when we are outside of class, inside we don't need it

Reply:
Everywhere I saw for constructor is ClassName::ClassName(). As I remember even VSC underline it if I did not use ClassName::ClassName(), but someone more experienced could clarify

Reply:
Yes, I mixed up everything... Give me few minutes to fix it